config/config.go

```diff
@@ -173,8 +174,31 @@ func GetLaunchTimeout() time.Duration {
 	return duration
 }
 
+// GetStopTimeout returns the grace period time for a pod to die
+// Returns time in seconds as an integer
 func GetStopTimeout() int {
-	return GetConfig().GetInt(COMPOSE_STOP_TIMEOUT)
+	// value from the config file expressed as seconds in integer format
+	// Viper implicitly casts string to int too
+	if valInt := GetConfig().GetInt(COMPOSE_STOP_TIMEOUT); valInt != 0 {
```

Comment:
Can't you change the default also as string so that you need not do so many validations

Reply:
Regardless of the default, we can take this as a string. However, if I do a GetString(), I'll need a strconv.Atoi() as well. Should I do similar to GetLaunchTimeout() ?

Reply:
what i meant was set the default as `10s` and you can just get using config.getDuration() right?

Reply:
In that case, https://github.com/paypal/dce-go/blob/95b7badfcd238958dbbdc99c84950bde1d2bf3b5/config/config.yaml#L20 will have to be changed to 20s. Also, the config under all the plugins need to be changed too.

Reply:
@kmala @JiaminZhu Should I set the default for clean pod timeout as duration everywhere? E.g: '20s' in stead of '20'

Reply:
my suggestion is to go with duration everywhere so its clean and simple